src/UI/Panels/ClipboardPanel.cs

```diff
@@ -4,8 +4,9 @@ namespace UnityExplorer.UI.Panels
 {
     public class ClipboardPanel : UEPanel
     {
-        public static object Current { get; private set; }
-
+        public static List<object> ClipboardItems = new List<object>();
+        public static int selectedItem = 0;
+        public static ClipboardPanel Instance;
```

Comment:
I don't think it needs to be a singleton.

Reply:
The reason I have this be a singleton is so then I can redraw the UI from static methods. For example, the Copy() method is static, because we want to be able to call that from the C# Console, but the UI elements are not static. The instance is just so we can redraw the UI from a static method.
If this needs to be changed, just lemme know and I can figure something out.